adapters/epics.py

```diff
@@ -68,9 +69,14 @@ def doProcess(self, dt):
 
 
 class EpicsAdapter(Adapter):
-    def run(self, target, bindings, *args, **kwargs):
+    def _parseArguments(self, arguments):
+        parser = ArgumentParser(description="Adapter to expose a device via EPICS")
+        parser.add_argument('-p', '--prefix', help='Prefix to use for all PVs', default='')
+        return parser.parse_args(arguments)
+
+    def run(self, target, bindings, prefix):
```

Comment:
This should be `doRun` now. I noticed pycharm is complaining about changing signature with respect to the base class. Not sure what the best solution is in this case. Maybe not defining `doRun` in the base class and instead doing the `if hasattr(self, 'doRun'): ...`-procedure? Did not see that coming, but it somehow makes sense that a method overriding a base class method should have the same signature.